config/config.go

```diff
@@ -121,6 +187,26 @@ type Log struct {
 	RecordPlanInSlowLog uint32 `toml:"record-plan-in-slow-log" json:"record-plan-in-slow-log"`
 }
 
+func (l *Log) getDisableTimestamp() bool {
```

Comment:
Do we need `getEnableTimestamp` which maybe more intuitive?

Reply:
Nope, in the internal logic we only need one option, and it is enough to have `getDisableTimestamp`. Historically we only have `DisableTimestamp`, so I created this mock function to provide a proxy for the added `EnableTimestamp` option.

Reply:
History can be changed by your hands.

Reply:
The internal logic is designed to support `Disable-xx`, of course I could make them all `Enable-xx` but I'm not sure about the consequences. 

![image](https://user-images.githubusercontent.com/29735669/67670633-4bee1d00-f9af-11e9-95f7-b9b9fc3f92ec.png)

If you insist on this, I could make another PR to fix this.

Reply:
The internal logic is not uniform. I think use `enable-xx` is more intuitive.